### PR TITLE
Update lan_admin.php

### PR DESCRIPTION
**Pull request by yesszus**

### CODE REVIEW

**Comment by yesszus** [comment]
@CaMer0n pls review this.

File: e107_admin/mailout.php [hunk 1]
```diff
@@ -283,7 +283,7 @@ class mailout_main_ui extends e_admin_ui
 			'mail_source_id' 		=> array('title' => LAN_MAILOUT_137, 'width' =>'5%', 'thclass' => 'center', 'class'=>'center', 'forced' => TRUE),
 			
 			'mail_selectors'		=> array('title' => LAN_MAILOUT_03, 'type'=>'method', 'data'=>false, 'nolist' => true, 'writeParms'=>'nolabel=0'),
-			'mail_title' 			=> array('title' => LAN_TITLE, 'type'=>'text', 'forced' => TRUE, 'data'=>'str', 'inline'=>true, 'writeParms'=>'size=xxlarge&required=1', 'help'=>''),
+			'mail_title' 			=> array('title' => LAN_MAILOUT_140, 'type'=>'text', 'forced' => TRUE, 'data'=>'str', 'inline'=>true, 'writeParms'=>'size=xxlarge&required=1', 'help'=>''),
```
Comment (CaMer0n) [inline e107_admin/mailout.php]: Just wondering why "Title" was changed to "Recipient Email" ?

File: e107_admin/mailout.php [hunk 2]
```diff
@@ -1586,7 +1586,7 @@ function options($parms, $value, $id, $attributes)
 			$preview = e_SELF."?mode=main&action=preview&id=".$id;
 			$text .= "<span class='btn-group'>";
 			$text .= "<a href='".$link."' class='btn btn-default' title='".LAN_MAILOUT_08."'>".E_32_MAIL."</a>";
-			$text .= "<a rel='external' class='btn btn-default e-modal' data-modal-caption='Email preview' href='".$preview."'  title='".LAN_PREVIEW."'>".E_32_SEARCH."</a>";
+			$text .= "<a rel='external' class='btn btn-default e-modal' data-modal-caption='Email ".LAN_PREVIEW."' href='".$preview."'  title='".LAN_PREVIEW."'>".E_32_SEARCH."</a>";
```
Comment (CaMer0n) [inline e107_admin/mailout.php]: This can just be changed to LAN_PREVIEW. 
`data-modal-caption='".LAN_PREVIEW."'  `

File: e107_admin/mailout.php [hunk 3]
```diff
@@ -1772,7 +1772,7 @@ function options($parms, $value, $id, $attributes)
 		$preview = e_SELF."?mode=main&action=preview&id=".$eid.'&user='.$user;
 
 		$text = "<span class='btn-group'>";
-		$text .= "<a rel='external' class='btn btn-default e-modal' data-modal-caption='Email preview' href='".$preview."' class='btn' title='Preview'>".E_32_SEARCH."</a>";
+		$text .= "<a rel='external' class='btn btn-default e-modal' data-modal-caption='Email ".LAN_PREVIEW."' href='".$preview."' class='btn' title='".LAN_PREVIEW."'>".E_32_SEARCH."</a>";
```
Comment (CaMer0n) [inline e107_admin/mailout.php]: The same for this one.

Comment (yesszus) [inline e107_admin/mailout.php]: Done!

**Pull request closed without merging**